src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isMale = sex === 'm';
+  const isMarriedMale = isMale && isMarried;
+  const isMarriedFemale = !isMale && isMarried;
+
+  let partnerTitle;
+
+  if (isMarriedMale || isMarriedFemale) {
+    partnerTitle = isMale ? 'wife' : 'husband';
+  } else {
+    partnerTitle = 'not married';
+  }
```

Comment:
- `partnerTitle = 'not married';` it doesn't make sense as you don't show a parent title if a person isn't married
- could be simplified just with `const partnerTitle = isMale ? 'wife' : 'husband';`